city_scrapers/spiders/cook_north_shore_mosquito.py

```diff
@@ -0,0 +1,98 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookNorthShoreMosquitoSpider(CityScrapersSpider):
+    name = "cook_north_shore_mosquito"
+    agency = "Cook County North Shore Mosquito Abatement District"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.nsmad.com"]
+    start_urls = ["https://www.nsmad.com/news-events/board-meetings/"]
+    TAG_RE = re.compile(r'<[^>]+>')
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        calendar = response.xpath('//*[@id="post-68"]/div/ul/li/ul')
+        for item in calendar.extract()[0].split('<li>')[1:]:
+            meeting = Meeting(
+                title=self._parse_title(),
+                description='',
+                classification=self._parse_classification(),
+                start=self._parse_start(item, response),
+                all_day=False,
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self):
+        """Parse or generate meeting title."""
+        return "Board of Trustees"
+
+    def _parse_classification(self):
+        """Parse or generate classification from allowed options."""
+        return BOARD
+
+    def _parse_start(self, item, response):
+        """Parse start datetime as a naive datetime object."""
+        year = '2019'
+        calendar_title = response.xpath('//*[@id="post-68"]/div/h2[1]').extract()[0]
+        if '2019' not in calendar_title:
```

Comment:
It looks like the header will presumably always include the year, so we should parse it out directly by looking for a four digit number rather than throwing an error when the year changes